<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Company Info</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>911</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabCompany">
     <attribute name="title">
      <string>Компания</string>
     </attribute>
     <widget class="QLabel" name="LCompanyName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Default Company</string>
      </property>
     </widget>
     <widget class="QLabel" name="LCEOPerson">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>451</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="LCEO">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>451</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Генеральный директор</string>
      </property>
     </widget>
     <widget class="QLabel" name="LCountEmpl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Штат:</string>
      </property>
     </widget>
     <widget class="QLabel" name="LCountEmplPerson">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Штат</string>
      </property>
     </widget>
     <widget class="QLabel" name="LCountEmpl_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Суммарная заработная плата сотрудников:</string>
      </property>
     </widget>
     <widget class="QLabel" name="LSumSalary">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>150</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Штат</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Сотрудники</string>
     </attribute>
     <widget class="QListWidget" name="listWidgetPerson">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>321</width>
        <height>541</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="LPersName">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>531</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Default Company</string>
      </property>
     </widget>
     <widget class="QLabel" name="LPersPosition">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>60</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>60</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Должность</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>80</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>С</string>
      </property>
     </widget>
     <widget class="QLabel" name="LPersPositionDate">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>80</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>110</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Оклад</string>
      </property>
     </widget>
     <widget class="QLabel" name="LPersSalary">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>110</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditPositions">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>230</y>
        <width>491</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>200</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>История должностей</string>
      </property>
     </widget>
     <widget class="QLabel" name="LPersSubLab_4">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>410</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Подчиненные</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditSub">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>440</y>
        <width>491</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="LPersSubLab">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>340</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Подчиненных</string>
      </property>
     </widget>
     <widget class="QLabel" name="LPersSubCount">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>340</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="LPersSayaryLOS">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>150</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>150</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>За выслугу</string>
      </property>
     </widget>
     <widget class="QLabel" name="LPersSalarySub">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>170</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>170</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>За подчиненных</string>
      </property>
     </widget>
     <widget class="QLabel" name="LPrecentFSType_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>130</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Из него:</string>
      </property>
     </widget>
     <widget class="QLabel" name="LPersSubCountFLevel">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>380</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="LPersSubLab_3">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>380</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Первого уровня</string>
      </property>
     </widget>
     <widget class="QLabel" name="LPersSubLab_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>360</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Из них:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Должности</string>
     </attribute>
     <widget class="QListWidget" name="listWidgetPosition">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>321</width>
        <height>541</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Должность:</string>
      </property>
     </widget>
     <widget class="QLabel" name="LNamePositionComent">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>60</y>
        <width>501</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Генеральный директор</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>100</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Всего в организации:</string>
      </property>
     </widget>
     <widget class="QLabel" name="LCountInCompany">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>100</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>150</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Базовая ставка:</string>
      </property>
     </widget>
     <widget class="QLabel" name="LRate">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>150</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>190</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Надбавка за год:</string>
      </property>
     </widget>
     <widget class="QLabel" name="LPrecentFY">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>190</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>280</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Надбавка за подчиненных:</string>
      </property>
     </widget>
     <widget class="QLabel" name="LPrecentFS">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>280</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="LPrecentFSType">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>300</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Только за подчиненных первого уровня</string>
      </property>
     </widget>
     <widget class="QLabel" name="LNamePosition">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>10</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>CEO</string>
      </property>
     </widget>
     <widget class="QLabel" name="LPrecentFYMAX">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>230</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>230</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Максимальная надбавка:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
